collect_app/src/main/java/org/odk/collect/android/activities/InstanceChooserList.java

```diff
@@ -93,7 +93,7 @@ private void init() {
         setupAdapter();
         instanceSyncTask = new InstanceSyncTask(settingsProvider);
         instanceSyncTask.setDiskSyncListener(this);
-        instanceSyncTask.execute();
+//        instanceSyncTask.execute();
```

Comment:
What's the next step here after disabling the task?

Reply:
There's an issue filed [here](https://github.com/getodk/collect/issues/4474). Sorry I should mentioned that in the PR description.